File: Skillrack/java/very easy/multiple of n till divisible by x.java
import java.util.*;
public class Hello {

    public static void main(String[] args) {
	Scanner sc=new Scanner(System.in);
	int n=sc.nextInt();
	int x=sc.nextInt();
	int multipe=n;
	boolean run=true;
	while(run){
	    System.out.print(multiple+" ");
	    if(multiple%x==0){
	        run=false;
	    }
	    multiple+=n;
	}

	}
}

File: Skillrack/C/very easy/part004/multiple of n till divisible by x.c
#include<stdio.h>
#include<stdlib.h>

int main()
{
int n,x;
scanf("%d%d",&n,&x);
int multiple=n;
int run=1;
while(run){
    printf("%d ",multiple);
    if(multiple%x==0){
        run=0;
    }multiple+=n;
}
}
